Content/0.4.5 - Preferences/preferences.json

```diff
@@ -2,8 +2,8 @@
     "preferences_title": "Preferences",
     "preferences_depositGuarantees_title": "Default Deposit Guarantees",
     "preferences_depositGuarantees_subtitle": "Set your guaranteed minimum for estimated deposits",
-    "preferences_hiddenEntities_title": "Hidden Accounts and Personas",
-    "preferences_hiddenEntities_subtitle": "Manage hidden Accounts and Personas",
+    "preferences_hiddenEntities_title": "Hidden Personas &amp; Accounts",
```

Comment:
why not use `&` directly? We are already doing so for example [here](https://github.com/radixdlt/apps-localization-src/blob/main/Content/0.4%20-%20Settings/settings.json#L5)

Reply:
Android IDE suggest to use escaped version in strings file on Android, I just done it automatically without giving it a thought. But it looks that crowdin does that conversion, so I will change it to `&`